enterprise/reporting/reporting.go

```diff
@@ -589,6 +589,10 @@ func (r *DefaultReporter) Report(ctx context.Context, metrics []*types.PUReporte
 			metric = transformMetricForPII(metric, getPIIColumnsToExclude())
 		}
 
+		if len(metric.StatusDetail.EventName) > 100 {
+			metric.StatusDetail.EventName = "INVALID_EVENT_NAME"
```

Comment:
Is truncation a better option instead of sending `INVALID_EVENT_NAME`?

Reply:
we looked at the data, most of them are stringified JSONs, URLs, and decided to go with this. Attached analysis doc in linear as a comment

Reply:
@achettyiitr One reason to not truncate is cardinality. Since these are mostly junk event names, if we truncate, these are going increase cardinality of event names on reporting